pco/src/data_types/floats.rs

```diff
@@ -435,17 +413,16 @@ mod tests {
     );
   }
 
-  // TODO fix this; float mult overestimates its bits saved
-  // #[test]
-  // fn test_choose_quant_mode() {
-  //   let lowest_num_bits = 1.0_f64.to_bits();
-  //   let k = 20;
-  //   let nums = (0..1000)
-  //     .map(|i| f64::from_bits(lowest_num_bits + (i << k)))
-  //     .collect::<Vec<_>>();
-  //   let (mode, _) = choose_mode_and_split_latents(&nums, &ChunkConfig::default()).unwrap();
-  //   assert_eq!(mode, Mode::FloatQuant(k));
-  // }
```

Comment:
How did this test get fixed?  I'm not seeing it, but if the PR contains code that fixes the failing test, can we add that information to the PR description?

Reply:
I'm being confusing: this was actually fixed in https://github.com/mwlon/pcodec/pull/219, and I should have uncommented the test there, but forgot.